### PR TITLE
Correct markdown in procedures

### PR DESCRIPTION
**Pull request by cbudz**
This PR fixes the markdown used in the procedure and removes the Additional Resources template text.

Preview here: http://file.rdu.redhat.com/cbudzilo/worker/build/tmp/en-US/html-single/

### CODE REVIEW

File: downstream/modules/catalog/proc-preparing-worker-installer.adoc [hunk 1]
```diff
@@ -36,38 +36,38 @@ h| servicescatalog_no_proxy |
 
 . Navigate to the {PlatformNameShort} setup or setup bundle installer directory.
```
Comment (lmalivert) [inline downstream/modules/catalog/proc-preparing-worker-installer.adoc:37]: According to IBM style guide, use "set up" as two words when using it as a verb. 
Suggestion: Navigate to the {PlatformNameShort} setup or setup bundle installer directory.  -> Navigate to the {PlatformNameShort} setup or set up the bundle installer directory.

File: downstream/modules/catalog/proc-preparing-worker-installer.adoc [hunk 2]
```diff
@@ -76,13 +76,14 @@ servicescatalog_controller_name: Marketing
...
 . Save each file
 . Open the `inventory` file in the {PlatformNameShort} installer:
 +
-====
+-----
 # vi inventory
-====
+-----
 +
 . Add the `host_vars` for each host under `[servicecatalog_workers]`
```
Comment (lmalivert) [inline downstream/modules/catalog/proc-preparing-worker-installer.adoc:88]: Suggestion: Add a period at the end of the sentence.

Comment (lmalivert) [reply downstream/modules/catalog/proc-preparing-worker-installer.adoc:88]: @cbudz Aside from 3 minor suggestions. LGTM.

File: downstream/modules/catalog/proc-preparing-worker-installer.adoc (hunk 1 above)
Comment (cbudz) [reply downstream/modules/catalog/proc-preparing-worker-installer.adoc:37]: hey @lmalivert 'setup' and 'setup bundle' are the names of the AAP  installers which is why they appear as they do here.

File: downstream/modules/catalog/proc-preparing-worker-installer.adoc (hunk 2 above)
Comment (cbudz) [reply downstream/modules/catalog/proc-preparing-worker-installer.adoc]: @lmalivert I've implemented your suggestions here and updated the preview.

Comment (lmalivert) [reply downstream/modules/catalog/proc-preparing-worker-installer.adoc]: @cbudz LGTM. Ready to merge.

**Pull request merged**